odoo/modules/module.py

```diff
@@ -401,19 +403,25 @@ def is_really_module(name):
             for mname in MANIFEST_NAMES:
                 if os.path.isfile(opj(dir, name, mname)):
                     return True
-        return [
-            clean(it)
+        return {
+            clean(it): os.sep.join([ad, it])
```

Comment:
It works by change, alternatively we rename `dir` => `ad` function wide
```suggestion
            clean(it): os.sep.join([dir, it])
```